test/snippet/search/bi_fm_index_cursor_to_rev_cursor.cpp

```diff
@@ -10,21 +10,22 @@ int main()
 
     seqan3::debug_stream << "Example to_rev_cursor()\n";
 
-    std::vector<seqan3::dna4> genome{"GAATTAACGAAC"_dna4};
-    seqan3::bi_fm_index index{genome};                         // build the bidirectional index
+    seqan3::dna4_vector genome{"GAATTAACGAAC"_dna4};
+    seqan3::bi_fm_index index{genome};                              // build the bidirectional index
 
-    auto cur = index.cursor();                                 // create a cursor
-    cur.extend_right("AAC"_dna4);                              // search the sequence "AAC"
-    seqan3::debug_stream << cur.path_label(genome) << '\n';    // outputs "AAC"
-    auto uni_it = cur.to_rev_cursor();                         // unidirectional cursor on the text "CAAGCAATTAAG"
-    seqan3::debug_stream << uni_it.path_label(genome) << '\n'; // outputs "CAA"
+    auto cur = index.cursor();                                      // create a cursor
+    cur.extend_right("AAC"_dna4);                                   // search the sequence "AAC"
+    seqan3::debug_stream << cur.path_label(genome) << '\n';         // outputs "AAC"
+    auto uni_it = cur.to_rev_cursor();                              // unidirectional cursor on the text "CAAGCAATTAAG"
+    auto genome_rev = genome | std::views::reverse;                 // create reverse text
+    seqan3::debug_stream << uni_it.path_label(genome_rev) << '\n';  // outputs "CAA"
```

Comment:
When I copy and paste the code, it prints:
```
Example to_rev_cursor()
AAC
CAA
GAAT
2
```
And that is, what it should do right?
If you remove the ```genome_rev```, than it prints:
```
Example to_rev_cursor()
AAC
GAA
GAAT
2
```
Because the reverse cursor is finding CAA at the end of the reversed text, but path_label was printing the end of the not-reversed text. (Path label used the wrong input.)

Reply:
Ah I just executed the original snippet; but you changed it. And I think you are right :)